server/app/views/api/ApiDocsView.java

```diff
@@ -278,9 +278,7 @@ private DivTag getNotes(Http.Request request) {
                 + " Static content questions are not shown on the API Response Preview because"
                 + " they do not include answers to questions."));
 
-    Accordion accordion = new Accordion().setTitle("How does this work?");
-    accordion.addContent(notesTag);
-
-    return accordion.getContainer();
+    return AccordionFactory.buildAccordion(
```

Comment:
question (non-blocking): why rename this to AccordionFactory?

Reply:
I found it misleading to have a utility class called Accordion that, when instantiated via the constructor, didn't provide you with an accordion component.  It could have been easy for someone not to realize that they had to call `getContainer`.  I also didn't like the name "container" because, to me, what was meant by "container" was the actual accordion element. 

I feel that it's clearer now.  If you need an accordion, you call AccordionFactory.buildAccordion().  Yes, it isn't following the factory design pattern, but it's a simple factory class.

Reply:
That makes sense to me! Thanks for explaining.